content/glossary/git.md

```diff
@@ -23,3 +20,23 @@ ${type} - Git Object Type as a string
 ${size} - size in bytes of ${content} represented as a string base 10.
 
 ${content} - []byte of the content
+
+# Git Blob
+
+A git `blob` is the type used for file contents in git:
```

Comment:
I like this. Having the reference here helps.

I also feel like we should disambiguate `blob`: "binary large object"

Reply:
I believe these are both resolved in the most recent update.